configure.ac

```diff
@@ -24,6 +24,13 @@ if test "x$enableval" != xyes ; then
   build_gifdiff=no
 fi)
 
+build_threaded=yes
+AC_ARG_ENABLE(threaded,
+[  --disable-threaded      do not build gifsicle with resize threading ability],
```

Comment:
Let's default this option to `maybe`, which means "yes if -lpthread exists, no otherwise".